docs/source/examples/QartodTestExample_Glider.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    e.dataset_id = "nrt_SEA067_M37"
</pre><p>We should probably save this dataset somewhere so we can "freeze" this example. I guess it may change over time in the server.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ioos/ioos_qc/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Sure, where should the dataset be uploaded? It's a netCDF around 4 MB

Reply:
We can probably upload as an asset in a GitHub release. Not the best solution though. Or Maybe @kwilcox have a better place for it in an ERDDAP server were folks here have more control.

Reply:
At 4MB I don't mind including it in the repo to avoid friction. Uploading as an asset and downloading in the notebook works too!

Reply:
I've reduced it to 480K and added it directly to the repo. It's a similar size to the pc02 example data already included